.authors.yml

```diff
@@ -424,47 +443,52 @@
   num_commits: 5
   first_commit: 2015-11-11 20:35:56
   github: rafaelhbarros
-- name: rulerofthehuns
+- name: Attila Nagy
   email: 58312016+rulerofthehuns@users.noreply.github.com
+  github: rulerofthehuns
+  aliases:
+  - rulerofthehuns
   num_commits: 1
   first_commit: 2019-12-03 15:22:09
-  github: rulerofthehuns
 - name: Bryan Van de Ven
   email: bryanv@continuum.io
-  num_commits: 13
+  num_commits: 500
   first_commit: 2012-10-17 04:24:05
-  alternate_emails:
-  - bryan@ubuntu.(none)
   aliases:
   - bryanv@continuum.io
-- name: srossross
+  - Bryan Van de Ven
+  alternate_emails:
+  - bryanv@continuum.io
+  - bryan@ubuntu.(none)
```

Comment:
Should this email address be `bryan@ubuntu.com` (or should we delete this line)?

Reply:
Ditto